appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/appellant-case.test.js

```diff
@@ -135,6 +135,67 @@ describe('appellant-case', () => {
 			expect(notificationBannerElementHTML).toContain('LPA application reference updated</p>');
 		});
 
+		it('should render a "Application decision date removed" notification banner when the application decision date is removed', async () => {
+			const appealId = appealData.appealId;
+			const appellantCaseId = appealData.appellantCaseId;
+			const validData = {
+				'application-decision-radio': 'no'
+			};
+
+			nock('http://test/')
+				.patch(`/appeals/${appealId}/appellant-cases/${appellantCaseId}`)
+				.reply(200, validData);
+
+			await request
+				.post(`${baseUrl}/${appealId}/appellant-case/application-decision-date/change`)
+				.send(validData);
+
+			const response = await request.get(`${baseUrl}/1${appellantCasePagePath}`);
+
+			const element = parseHtml(response.text);
+
+			expect(element.innerHTML).toMatchSnapshot();
```

Comment:
Did you mean to have this too? Means this test includes a full-page snapshot. Applies to the one below as well.